src/jit/gentree.cpp

```diff
@@ -14397,7 +14397,7 @@ bool Compiler::gtTreeHasSideEffects(GenTreePtr tree, unsigned flags /* = GTF_SID
     return true;
 }
 
-GenTreePtr Compiler::gtBuildCommaList(GenTreePtr list, GenTreePtr expr)
+GenTreePtr Compiler::gtBuildCommaList(GenTreePtr list, GenTreePtr expr DEBUGARG(bool remorphing))
```

Comment:
I'm not sure what you mean by "in cases that haven't been explicitly checked". It would seem to me that `fgGlobalMorph` would be a better all-around approach, but perhaps I'm not seeing the value of preserving it in a subset of remorphing cases.